requirements.txt

```diff
@@ -6,7 +6,7 @@ Jinja2>=2.11.2,<2.12
 mistune>=2.0.0a4,<2.1
 requests>=2.24,<3.0
 iso-639>=0.4.5,<0.5
-zimscraperlib>=1.3.6,<1.4
+zimscraperlib
```

Comment:
Please specify a fixed version (e.g., "==3.2.0", this is our new standard)

Reply:
and this too